apps/daimo-mobile/src/view/shared/shareSheetURL.tsx

```diff
@@ -0,0 +1,26 @@
+import { Alert, Platform, Share, ShareAction } from "react-native";
+
+export async function shareSheetURL(
```

Comment:
recommend eg

```ts
// Share via system share sheet = user picks messaging app
export async function shareURL(url: string) {
```
...comment, verb name, & the dollars argument looks unused

Reply:
done, thanks